### DISCUSSION TITLE
Wildcard/splat routes question

### DISCUSSION THREAD

**Discussion by BootstrapBilly** [TanStack/router#1185] [Q&A]
In my application, i have a route structure like so:

```
/viewOne
/viewTwo
/viewThree
```

On these routes, it is possible to perform the same action. 

For example:

```
/viewOne/edit/$id
/viewTwo/edit/$id
/viewThree/edit/$id
```

Using file based routing, is it possible to define the edit route once, and have it available in the child routes for viewOne, viewTwo and viewThree?

Perhaps something like:

`$/edit/$id`

**Comment by schiller-manuel** [comment]
no, currently not
a similar question is here: https://github.com/TanStack/router/discussions/1171
